src/api/api.ts

```diff
@@ -0,0 +1,56 @@
+
+export interface Good{
+  categoryTypeId: string,
+  id:string,
+  img:string,
+  label:string,
+  price:number,
+}
+export interface Category {
+   type:string,
+   label:string,
+   id:string,
+}
+
+export class Api {
+
+  endPoints={
+    goods:'/api/goods',
+    categories:'/api/categories',
+    popular_categories:'/api/popular_categories',
```

Comment:
```suggestion
    popularCategories:'/api/popular_categories',
```
мы camelCase используем в js